docs/build/program/apis/data-client.md

```diff
@@ -846,3 +846,232 @@ For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/
 
 {{% /tab %}}
 {{< /tabs >}}
+
+### CreateDataset
+
+Create a new dataset.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- name (str): The name of the dataset being created.
+- organization_id (str): The ID of the organization where the dataset is being created.
+
+**Returns:**
+
+- (`str`): The dataset ID of the created dataset.
```

Comment:
```suggestion
- [(str)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): The dataset ID of the created dataset.
```
We typically do not use monospace for the return types since it's a type, not a literal parameter name. Do include link though.